### PR TITLE
base cli tool

### PR DESCRIPTION
**Pull request by stevemcghee**
just a skeleton with some sample verbs but no implementations yet.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,3 @@
+module github.com/GoogleCloudPlatform/reliable-app-platforms
```
Comment (muncus) [inline go.mod:1]: Do we need a top-level go module? i dont think it is necessary. same for `go.work`

**Review by muncus — APPROVED** [review]
please remove the `rapctl` binary before merging.

and adding the `go.sum` will save us all an extra build step. without it, we'll need to run `go mod tidy` first.

File: go.mod (hunk 1 above)
Comment (stevemcghee) [reply go.mod:1]: without it, on `go build` i get:

```
❯ go build
cmd/buildPlatform.go:22:2: missing go.sum entry for module providing package github.com/spf13/cobra (imported by github.com/GoogleCloudPlatform/reliable-app-platforms/cli/rapctl/cmd); to add:
        go get github.com/GoogleCloudPlatform/reliable-app-platforms/cli/rapctl/cmd
```

**Review by muncus — APPROVED** [review]
Thanks for building the skeleton of our CLI! 🚀

**Pull request merged**